lib/utils/socketErrorMessages.js

```diff
@@ -94,6 +94,23 @@ const getSnippetLogs = ({testId, definitions, results, level, verbosity}, transl
 	}, []).filter(Boolean).join('\n');
 };
 
+const commandErrors = {
+	'notSupportedPlatform': () => t['commandError.notSupportedPlatform'](),
+	'notSupportedIL': () => t['commandError.notSupportedIL'](),
```

Comment:
Will be removed from the backend as obsolete code. Obsolete due to SUIT-14556. Consider removal of this code completely. @JeKachan @Noorama